workers/docker-worker/test/integration/disable_seccomp_containers_test.js

```diff
@@ -0,0 +1,115 @@
+const assert = require('assert');
+const settings = require('../settings');
+const DockerWorker = require('../dockerworker');
+const TestWorker = require('../testworker');
+
+suite('disableSeccomp capability', () => {
+  let worker;
+
+  setup(async () => {
+    settings.cleanup();
+  });
+
+  teardown(async () => {
+    settings.cleanup();
+    if (worker) {
+      await worker.terminate();
+    }
+    worker = null;
+  });
+
+  test('task error when necessary scopes missing', async () => {
+    settings.configure({
+      dockerConfig: {
+        disableSeccomp: true,
```

Comment:
Should this be `allowDisableSeccomp` instead?

Reply:
You're right. Thanks! I don't think that explains the missing scopes on the test task though.